streams/src/main/java/org/apache/kafka/streams/kstream/internals/graph/StreamsGraphNode.java

```diff
@@ -94,4 +107,12 @@ public InternalStreamsBuilder internalStreamsBuilder() {
 
     public abstract void writeToTopology(final InternalTopologyBuilder topologyBuilder);
 
+    @Override
+    public String toString() {
```

Comment:
Should we override this for all the subclasses so that we can have a more informative intermediate logical plan representation for debugging purposes?

Reply:
yep, I'll have to put some thought into what's relevant without being too spammy